drizzlepac/alignimages.py

```diff
@@ -918,46 +890,46 @@ def determine_fit_quality(imglist, filtered_table, catalogs_remaining, print_fit
         # if fit_rms_val < max_rms_val:
         #     fitRmsCheck = True
 
-        consistency_check = True
+        consistencyCheck = True
```

Comment:
Variable should not be camelcase. Rename 'consistencyCheck' to 'consistency_check' and find/replace the remaining instances.